modules/jlink/artifacts/opt/jboss/container/java/jlink/mkjreimage.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# TODO: Still Needed?
+set -euo pipefail
+
+depsfile="$S2I_JLINK_TEMP_PATH/module-deps.txt"
+
+function generate_jre_image() {
+	test -f $depsfile
+	modules="$(cat $depsfile)"
+
+	$JAVA_HOME/bin/jlink --output "$S2I_JLINK_OUTPUT_PATH" \
+    	   	--add-modules "$modules" \
+		--strip-debug --no-header-files --no-man-pages \
+		--compress=2
```

Comment:
```suggestion
		--compress=zip-6
```

From JDK 21 `jlink --help`:
```
      --compress <compress>             Compression to use in compressing resources:
                                        Accepted values are:
                                        zip-[0-9], where zip-0 provides no compression,
                                        and zip-9 provides the best compression.
                                        Default is zip-6.
                                        Deprecated values to be removed in a future release:
                                        0:  No compression. Equivalent to zip-0.
                                        1:  Constant String Sharing
                                        2:  Equivalent to zip-6.
```